src/libostree/libostree.sym

```diff
@@ -373,12 +373,10 @@ global:
  *                         NOTE NOTE NOTE
  */
 
-/* Remove comment when first new symbol is added
-LIBOSTREE_2016.XX {
+LIBOSTREE_2017.3 {
```

Comment:
Ah, indeed, I keep forgetting about the Debian stuff when it’s in a separate `debian-master` branch. My bad.

Since the customisation is within our Debian package of OSTree, we could add the new symbol to the Debian symbols file without causing upgrade problems in future. I’ll file a follow-up PR to do this when I get a chance.